cluster-scope/base/core/namespaces/ds-example-project/namespace.yaml

```diff
@@ -6,5 +6,8 @@ metadata:
     openshift.io/display-name: "Data Science: Example project"
     openshift.io/requester: data-science
     operate-first.cloud/issue: "https://github.com/operate-first/support/issues/76"
```

Comment:
Duplicate field on line 10?

Reply:
I'm a bit unsure about this myself, @4n4nd what do you think?

Reply:
the original label might be being used somewhere else, let's just add this one